lib/authenforce/middleware.go

```diff
@@ -1,21 +1,72 @@
 package authenforce
 
 import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"io"
+	"io/ioutil"
 	"net/http"
+	"sync/atomic"
 
+	"github.com/cuvva/cuvva-public-go/lib/cher"
 	"github.com/cuvva/cuvva-public-go/lib/crpc"
 	"github.com/wearemojo/mojo-public-go/lib/authparsing"
+	"github.com/wearemojo/mojo-public-go/lib/errgroup"
+	"github.com/wearemojo/mojo-public-go/lib/merr"
 )
 
-func (e Enforcer) CRPCMiddleware() crpc.MiddlewareFunc {
+func (e Enforcers) CRPCMiddleware() crpc.MiddlewareFunc {
```

Comment:
Wonder if this might benefit from being a function rather than a method on `Enforcers`? As we may move to other RPC setups in future, may want to split CRPC compatibility into another package in the future